french/screens/scroll/hunt_allies.rpy

```diff
@@ -4,45 +4,44 @@ translate french strings:
 
     # game/screens/scroll/hunt_allies.rpy:4
     old "She currently doesn't possess any advanced combat capabilities but nonetheless occupies a number of roles in the Royal Military: nurse, bartender, cook and most importantly, Field Operations Support for Hunters."
-    new ""
+    new "Elle ne possède actuellement aucune capacité de combat avancée mais occupe néanmoins un certain nombre de rôles au sein de l'armée royale : infirmière, barwoman, cuisinière et surtout, soutien aux opérations sur le terrain pour les chasseurs."
 
     # game/screens/scroll/hunt_allies.rpy:5
     old "Some people say that she haunts the servers of the Red Shark when in rest mode. If you try and access non-approved websites on the Network, {i}she will know{/i}."
-    new ""
+    new "Certains disent qu'elle hante les serveurs du Red Shark lorsqu'elle est en mode repos. Si vous essayez d'accéder à des sites Web non approuvés sur le réseau, {i}elle le saura{/i}."
 
     # game/screens/scroll/hunt_allies.rpy:7
     old "An unique existence on Remnant, Janara was originally created by the Fire God and Weser van Goethe as a possible receptacle for the former's soul and power. She was eventually deemed unsuitable for the process and in her newly assigned role as keeper for the sealed city of Peak, she met Team JPDE and their friends. Through repeat encounters, Janara Peach was then born through the love of her found family and friends."
-    new ""
+    new "Existence unique sur Remnant, Janara a été créée à l'origine par le Dieu du Feu et Weser van Goethe comme réceptacle possible de l'âme et du pouvoir du premier. Elle a finalement été jugée inadaptée au processus et à son rôle nouvellement assigné de gardienne de la ville scellée de Peak, elle a rencontré l'équipe JPDE et leurs amis. Grâce à des rencontres répétées, Janara Peach est alors née de l'amour de sa famille et de ses amis."
 
     # game/screens/scroll/hunt_allies.rpy:8
     old "Janara is a magic user, the only one known to mankind other than the fabled Maidens of Season. Her main application of it is controlling the metal known as pyrobsidian and creating a number of different creatures which obey all her commands. While she no longer thinks of Peak as just her city, she hasn't relinquished the role of keeper and protector: the scope has just been extended to everyone who now call the Duchy home."
-    new ""
+    new "Janara est une utilisatrice de magie, la seule connue de l'humanité autre que les légendaires Maidens of Season. Sa principale application est de contrôler le métal connu sous le nom de pyrobsidienne et de créer un certain nombre de créatures différentes qui obéissent à tous ses ordres. Bien qu'elle ne soit plus considère Peak comme sa ville, mais elle n'a pas abandonné son rôle de gardienne et de protectrice : son champ d'action a été étendu à tous ceux qui habitent désormais dans le Duché."
```

Comment:
Doit-on traduire "Maiden of Season" ?

Reply:
si tu veux, dans le glossaire j'ai mis "dames" donc je vais suivre ça